linking to the agent update command in this upgrade guide (#1809)
ngrok/ngrok-docs: agent/upgrade-v2-v3.mdx

sidebarTitle: Upgrade to v3
description: Learn the best way to upgrade from ngrok agent v2 to v3.
---
## Upgrading the ngrok Agent to the latest version 

Before upgrading, we highly encourage you to make sure you have the latest version of the ngrok agent. If you don't, please [update to the latest version](/agent/cli#ngrok-update) before proceeding.

The latest ngrok agent is packed with a ton of new capability but because it is a major release of the agent, there are also some breaking changes to watch out for.

## Breaking changes

<<<<<<< HEAD
### Upgrading the ngrok Agent to the latest version 

Before proceeding, please ensure that you've already [updated to the latest version](/agent/cli#ngrok-update) of the ngrok agent.

=======

> >>>>>>> 3516e1544 (changing to h2)

Reviewer: ```suggestion
```

## Breaking changes

### Upgrading the ngrok Agent to the latest version 

> Before upgrading, we highly encourage you to make sure you have the latest version of the ngrok agent. If you don't, please [update to the latest version](/agent/cli#ngrok-update) before proceeding.

Reviewer 2: We can tighten this up and also make Vale happy :)

```suggestion
Before proceeding, please ensure that you've already [updated to the latest version](/agent/cli#ngrok-update) of the ngrok agent.
```

Author: thought about doing that initially! Makes sense to me
